test/react16_suggestors/dependency_override_updater_test.dart

```diff
@@ -0,0 +1,370 @@
+                '  over_react: ^2.0.0\n'
+                '\n'
+                'dev_dependencies:\n'
+                '  dart_dev: ^2.0.1\n'
+                '\n'
+                'dependency_overrides:\n'
+                '  react:\n'
+                '    git:\n'
+                '      url: git@github.com:cleandart/react-dart.git\n'
+                '      ref: 5.0.0-wip\n'
+                '  over_react:\n'
+                '    git:\n'
+                '      url: git@github.com:Workiva/over_react.git\n'
+                '      ref: 3.0.0-wip\n'
+                '',
+            expectedOutput: ''
+                'dependencies:\n'
+                '  test: 1.5.1\n'
+                '  react: ^4.6.0\n'
+                '  over_react: ^2.0.0\n'
+                '\n'
+                'dev_dependencies:\n'
+                '  dart_dev: ^2.0.1\n'
+                '\n'
+                'dependency_overrides:\n'
+                '  react:\n'
+                '    git:\n'
+                '      url: https://github.com/cleandart/react-dart.git\n'
+                '      ref: 5.0.0-wip\n'
+                '  over_react:\n'
+                '    git:\n'
+                '      url: https://github.com/Workiva/over_react.git\n'
+                '      ref: 3.0.0-wip\n'
+                '',
+          );
+        });
+
+        test('git (HTTPS)', () {
+          testSuggestor(
+            expectedPatchCount: 2,
+            shouldDartfmtOutput: false,
+            input: ''
+                'dependencies:\n'
+                '  test: 1.5.1\n'
+                '  react: ^4.6.0\n'
+                '  over_react: ^2.0.0\n'
+                '\n'
+                'dev_dependencies:\n'
+                '  dart_dev: ^2.0.1\n'
+                '\n'
+                'dependency_overrides:\n'
+                '  react:\n'
+                '    git:\n'
+                '      url: https://github.com/cleandart/react-dart.git\n'
+                '      ref: 5.0.0-wip\n'
+                '  over_react:\n'
+                '    git:\n'
+                '      url: https://github.com/Workiva/over_react.git\n'
+                '      ref: 3.0.0-wip\n'
+                '',
+            expectedOutput: ''
+                'dependencies:\n'
+                '  test: 1.5.1\n'
+                '  react: ^4.6.0\n'
+                '  over_react: ^2.0.0\n'
+                '\n'
+                'dev_dependencies:\n'
+                '  dart_dev: ^2.0.1\n'
+                '\n'
+                'dependency_overrides:\n'
+                '  react:\n'
+                '    git:\n'
+                '      url: https://github.com/cleandart/react-dart.git\n'
+                '      ref: 5.0.0-wip\n'
+                '  over_react:\n'
+                '    git:\n'
+                '      url: https://github.com/Workiva/over_react.git\n'
+                '      ref: 3.0.0-wip\n'
+                '',
+          );
+        });
+
+        test('path', () {
```

Comment:
This test seems like it isn't testing anything.

Reply:
🤦‍♂ Yep, that is correct haha